exercises/ruby-2-a/README.md

```diff
@@ -1 +1,31 @@
-TODO
+## Introduction
+
+This is Part 2 of our research into how people write Ruby differently. This exercise should take 10-15 minutes for a programmer who is proficient in Ruby. There is no right or wrong way to approach it - just do what feels the most natural to you. Feel free to use books, Google or Stack Overflow, just like you would if you were programming normally, but please don't actively look for a solution to the problem as this will negatively affect the research.
+
+## Instructions
+
+Your task is to extend the existing shopping cart with a new functionality. Once the extension is added the shopping cart should be able to read and apply discount coupons to the current items. The supported coupons are:
```

Comment:
I would suggest to also include a description of the existing code here, to have the student get an idea what the current code does. See [this C# example](https://github.com/exercism/research_experiment_1/tree/master/exercises/csharp-2-b#instructions).